### ISSUE TITLE
Bloodlord Claymore - wrong stat calculation

### ISSUE THREAD

**Issue by Uluthefiregod** [WarEmu/WarBugs#16796]
**Expected behavior and actual behavior:**

The bloodlord claymore for knight should give you only 5% armor reduce, but it reduces armor by much more.

**Steps to reproduce the problem:**

I equiped first my "Nightless" claymore > 3664 Armor = 100%

Then i equiped the "bloodlord Claymore" > 3297 Armor = 89,98 %

It reduces armor by 11% looks like math behind is wrong? Should be 5% as tooltip?

**Testing Screenshots/Videos/Evidences (always needed):**

![image](https://user-images.githubusercontent.com/64554766/81818074-d8b48e80-952d-11ea-8b41-52f399871686.png)

![image](https://user-images.githubusercontent.com/64554766/81817980-b884cf80-952d-11ea-90eb-9aa916d6fe89.png)

How it is working its bassically useless for Tanks because of huge armor reduction, maybe change it also for tank class and take away defence reductions completly would make anyway more sence.

**Comment by Kaelang** [comment]
Have you taken into account the armor from the talis in your Dayless?

**Comment by Uluthefiregod** [comment]
What armor talis? There is none into the "Nightless" its STR and Tough. Check screenshot please.

**Comment by Kaelang** [comment]
> What armor talis? There is none into the "Nightless" its STR and Tough. Check screenshot please.

Yeah - I have my Str gear on, thinking I had armor gear on. Saw the same talis. My mistake.

Will tag up and look to confirm.

**Comment by Uluthefiregod** [comment]
No Problem, thanks for checking.

**Comment by Kaelang** [comment]
![image](https://user-images.githubusercontent.com/60117106/81819410-309bc700-9527-11ea-844d-5d93b1e1f438.png)
![image](https://user-images.githubusercontent.com/60117106/81819443-3b565c00-9527-11ea-9f15-5c61bc900c09.png)

Can confirm it's taking 10% as opposed to 5%.

**Comment by Vayra** [comment]
It takes 10% yes. Tested that a while back. It also doesn't stack if you have 2 equipped. Further, if you unequip 1 of them, it takes away the bonus entirely. I'll make a separate report about that now.

**Comment by GK111** [comment]
it was always like this

1h weapon- 5,7% armor reduction

2h - 11% armor reduction ( because double stats)

**Comment by Kaelang** [comment]
> it was always like this
> 
> 1h weapon- 5,7% armor reduction
> 
> 2h - 11% armor reduction ( because double stats)

See my 1 hander having the same effect.
